doc/neogen.txt

```diff
@@ -150,8 +150,8 @@ Neogen will go until the start of the function and start annotating for you.
 
 Parameters~
 {opts} `(table)` Optional configs to change default behaviour of generation.
- - {opts.type} `(string, default: "func")` Which type we are trying to use for generating annotations.
-   Currently supported: `func`, `class`, `type`, `file`
+ - {opts.type} `(string, default: "any")` Which type we are trying to use for generating annotations.
+   Currently supported: `any`, `func`, `class`, `type`, `file`
```

Comment:
It is recommended to change annotations in `init.lua` responsible of this function annotation